azext_iot/tests/iothub/devices/test_iothub_device_tracing.py

```diff
@@ -42,22 +35,24 @@ def test_iothub_device_distributed_tracing(self):
 
             self.cmd(
                 self.set_cmd_auth_type(
-                    f"iot hub device-identity create -d {device_ids[0]} -n {LIVE_HUB} -g {LIVE_RG}",
+                    f"iot hub device-identity create -d {device_ids[0]} -n {self.entity_name} -g {self.entity_rg}",
                     auth_type=auth_phase,
                 )
             )
 
             self.cmd(
                 self.set_cmd_auth_type(
-                    f"iot hub distributed-tracing show -d {device_ids[0]} -n {LIVE_HUB} -g {LIVE_RG}",
+                    f"iot hub distributed-tracing show -d {device_ids[0]} -n {self.entity_name} -g {self.entity_rg}",
                     auth_type=auth_phase,
                 ),
                 checks=self.is_empty(),
             )
 
             result = self.cmd(
                 self.set_cmd_auth_type(
-                    f"iot hub distributed-tracing update -d {device_ids[0]} -n {LIVE_HUB} -g {LIVE_RG} --sm on --sr 50",
+                    "iot hub distributed-tracing update -d {} -n {} -g {} --sm on --sr 50".format(
```

Comment:
I'm curious why we have to refactor out of f strings here.

Reply:
The line of code was exceeding 130 characters, but we can have multiline f strings

Reply:
Should I change these back to f strings?

Reply:
Please